ports/mujs/portfile.cmake

```
@@ -9,6 +9,7 @@ vcpkg_from_github(
 )
 
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
+file(COPY ${CMAKE_CURRENT_LIST_DIR}/mujsConfig.cmake.in DESTINATION ${SOURCE_PATH})
 
 vcpkg_configure_cmake(
   SOURCE_PATH ${SOURCE_PATH}
```

```suggestion
vcpkg_cmake_configure(
  SOURCE_PATH "${SOURCE_PATH}"
```
Please also remove the `PREFER_NINJA`.